fuzzing/oss_fuzz_build.sh

```diff
@@ -10,6 +10,9 @@ do
   ALL_JARS="$ALL_JARS $(basename $jarfile)"
 done
 
+# Copy sample config file for fuzzer
+cp $SRC/sigstore-java/sigstore-java/src/test/resources/dev/sigstore/oidc/server/config.json $OUT/oidc-config.json
```

Comment:
hrmm.. I missed this, but this should probably be done in gradle build file.

in fuzzing/build.gradle.kts
```
tasks.register<Copy>("copyToFuzzOut") {
    dependsOn(tasks.build)
    into(project.findProperty("fuzzOut") ?: project.layout.buildDirectory.dir("fuzzOut"))
    from(sourceSets.main.get().runtimeClasspath)
    // add this:
    from(project(":sigstore-java").layout.projectDirectory.dir("src/test/resources/dev/sigstore/oidc/server/config.json"))
    rename("config.json", "oidc-config.json")
}

}
```

Reply:
Done